Samples/JavaKitSampleApp/Sources/JavaKitExample/JavaKitExample.swift

```diff
@@ -27,7 +27,10 @@ extension HelloSwift: HelloSwiftNativeMethods {
     let answer = self.sayHelloBack(i + j)
     print("Swift got back \(answer) from Java")
 
+#if !canImport(Android)
+    // no class variables in Kotlin
```

Comment:
But Android != Kotlin, there's Java there too right? the comment strikes me as bit weird, is this really necessary?

Reply:
The sample Android app is written in Kotlin. I do not have the cycles to rewrite it in Java.

Reply:
Maybe I'm confused, this is the JavaKitExample, why does what the android sample app matter to this example project?